CHANGELOG.md

```diff
@@ -118,7 +118,6 @@ marked with [x] are considered complete.
 - [ ] Caching strategies in DomainNegotiator
 - [ ] Caching strategies in DomainConfigOverrides
 - [ ] Cache in the DomainAccessManager
-- [ ] Add filter options to domain_access and domain_source views
```

Comment:
This was already committed to the GitHub version, which hasn't been pushed to d.o. yet because other changes drop support for Drupal 8.4.